app/components/OutcomeBar.tsx

```diff
@@ -37,6 +39,8 @@ type OutcomeTokenMarginalPricesResponse = {
 };
 
 export const OutcomeBar = ({ market }: OutcomeBarProps) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasOutcomePercentages, setHasOutcomePercentages] = useState(false);
   const { id } = market;
 
   const { data: trade } = useQuery({
```

Comment:
can you check if we can use `isLoading` from react query?

```suggestion
  const { data: trade, isLoading } = useQuery({
```